doc/usage.rst

```diff
@@ -138,6 +138,39 @@ Alternative hashing algorithms supported by :mod:`hashlib` can be used if necess
     print(pooch.file_hash("data/c137.csv", alg="sha512"))
 
 
+Bypassing the hash check
+------------------------
+
+Sometimes we might not know the hash of the file or it could change on the server
+periodically. In these cases, we need a way of bypassing the hash check.
+One way of doing that is with Python's ``unittest.mock`` module, and setting the
+hash value to ``ANY`` (a special object which passes all equality tests).
+
+In this example, we are downloading a list of weather stations around Australia:
+
+.. code:: python
+
+    import datetime
+    from unittest import mock  
+    import pooch
+
+    CURRENT_DATE = datetime.datetime.now().date()
+    
+    GOODBOY = pooch.create(
+        path=pooch.os_cache("bom_daily_stations") / CURRENT_DATE,
+        base_url="ftp://ftp.bom.gov.au/anon2/home/ncc/metadata/sitelists/",
+        # Use `mock.ANY` for the hash value if you do not care that it changes
+        registry={
+            "stations.zip": mock.ANY,
+        },
+    )
+
+Pooch will fetch the files **once** per day, even if run multiple times. If you omit ``CURRENT_DATE`` from the path,
+then Pooch will only fetch the files once, unless they are deleted from the cache.
```

Comment:
```suggestion
Because hash check is always ``True``, Pooch will only download the file once. When running 
again at a different date, the file will be downloaded again because the local cache folder 
changed and the file is no longer present in it. If you omit ``CURRENT_DATE`` from the cache 
path, then Pooch will only fetch the files once, unless they are deleted from the cache.
```